lib/fs.js

```diff
@@ -1046,6 +1046,9 @@ fs.chownSync = function(path, uid, gid) {
 // converts Date or number to a fractional UNIX timestamp
 function toUnixTimestamp(time) {
   if (typeof time === 'number') {
+    if (!isFinite(time)) {
```

Comment:
Should use [`Number.isFinite()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite) instead of `isFinite()`:

> In comparison to the global isFinite() function, this method doesn't forcibly convert the parameter to a number. This means only values of the type number, that are also finite, return true.

Doesn't matter here, but good for consistency.